packages/reports/addon/components/report-builder.hbs

```diff
@@ -43,9 +43,20 @@
             />
           </button>
           {{#unless @isFiltersCollapsed}}
-            <span class="report-builder__container-header__message flex align-items-center">
-              Click <DenaliIcon @icon="filter-add" @size="small" class="report-builder__container-header__message--icon m-x-2" /> next to a dimension/metric to add a filter.
-            </span>
+            <DenaliInputGroup @label="Row Limit" class="report-builder__row-limit">
+              {{#if (gt this.request.limit 10000)}}
```

Comment:
We might need to change this to a warning about incomplete results because the UI doesn't support paging through fact requests

Reply:
For the time being, I think we should treat 10k the upper limit

Reply:
At least for the UI

Reply:
Yeah seems like there is some issues we'd have to fix to support above 10k well.